src/css/profile/wearable/changeable/common/circularindexscrollbar.less

```diff
@@ -34,7 +34,7 @@
 		width: 100%;
 		height: 100%;
 		text-align: center;
-		font-size: 70 * @unit_base;
+		font-size: 72 * @unit_base;
```

Comment:
According to pdf guide, this should be 80.

Reply:
The PDF guideline is outdated. This is value from latest reference.